src/transforms/interval.js

```diff
@@ -59,7 +49,13 @@ function maybeIntervalMidK(k, maybeInsetK, options) {
     ...options,
     [k]: {
       label: labelof(v),
-      transform: data => valueof(data, value).map(interval.mid)
+      transform: data => {
+        const V1 = valueof(data, value).map(v => interval.floor(v));
```

Comment:
It might not be safe to use valueof(data, value).map here if the channel is provided as a typed array and the interval returns something else, such as a date. I’ve added a commit that switches to Array.from.